sphinx_antsibull_ext/__init__.py

```diff
@@ -0,0 +1,42 @@
+# coding: utf-8
+# Author: Felix Fontein <felix@fontein.de>
+# License: GPLv3+
+# Copyright: Ansible Project, 2020
+'''
+Antsibull minimal Sphinx extension which adds some features from the Ansible doc site.
+'''
+
+from __future__ import (absolute_import, division, print_function)
+
+__version__ = "0.1.0"
+__license__ = "BSD license"
+__author__ = "Felix Fontein"
+__author_email__ = "felix@fontein.de"
+
+
+from .pygments_lexer import AnsibleOutputLexer
+from .assets import setup_assets
+
+
+def setup(app):
+    '''
+    Initializer for Sphinx extension API.
+    See http://www.sphinx-doc.org/en/stable/extdev/index.html#dev-extensions.
+    '''
+
+    # Add Pygments lexers
+    for lexer in [
+        AnsibleOutputLexer(startinline=True)
+    ]:
+        app.add_lexer(lexer.name, lexer)
+        for alias in lexer.aliases:
+            app.add_lexer(alias, lexer)
```

Comment:
This is a little too nested: you could've iterated over everything with a single loop
```python
lexer = ...
for name in (lexer.name, *lexer.aliases):
    app.add_lexer(name, lexer)
````

Reply:
#212